app/views/docs/view.html.erb

```diff
@@ -1,2 +1,49 @@
-
-<%= raw @result %>
+<section class="cli-header bg-light px-3 py-3 pt-md-5 pb-md-4 mx-auto border-bottom">
```

Comment:
CLI was capitalized, but when rendered it is "Cli"

![image](https://user-images.githubusercontent.com/8007722/71860759-13ab3f00-30c2-11ea-9800-4ed46af993c3.png)

Reply:
My bad here, I was using the directory structure and titleizing. Fixed for the upcoming push.